runtime/src/lib.rs

```diff
@@ -370,14 +370,14 @@ impl pallet_gear_scheduler::Config for Runtime {
     type BlockLimiter = GearGas;
     type ReserveThreshold = ConstU32<1>;
     type WaitlistCost = ConstU64<100>;
+    type MailboxCost = ConstU64<100>;
```

Comment:
How did you choose this value?

Reply:
Same as mailbox threshold. May be changed any time without need to update tests etc.